lumicks/pylake/force_calibration/tests/test_hydro.py

```diff
@@ -0,0 +1,241 @@
+            25e-9,
+            [1.018966397e-08, 8.174707470e-09, 1.538233969e-09, 6.041701478e-10, 3.637960655e-10],
+        ],
+        [
+            4800,
+            1.08,
+            14000,
+            0.4,
+            1.16e-8,
+            0.52e-6,
+            1030.0,
+            1060.0,
+            100e-9,
+            [8.943606396e-09, 7.491080241e-09, 1.661923005e-09, 6.693646549e-10, 4.056422851e-10],
+        ],
+        [
+            4800,
+            1.08,
+            14000,
+            0.4,
+            1.26e-8,
+            0.52e-6,
+            997.0,
+            1060.0,
+            100e-9,
+            [8.946161914e-09, 7.504194956e-09, 1.661270710e-09, 6.680646956e-10, 4.045687481e-10],
+        ],
+        [
+            2445,
+            0.542,
+            14000,
+            0.4,
+            1.978234e-8,
+            1.04e-6,
+            997.0,
+            1060.0,
+            100e-9,
+            [1.862354180e-08, 1.043633041e-08, 9.856086009e-10, 3.640796920e-10, 2.161805755e-10],
+        ],
+        [
+            1165,
+            0.25,
+            14000,
+            0.4,
+            4.2694e-8,
+            2.2e-6,
+            997.0,
+            1060.0,
+            100e-9,
+            [3.909336166e-08, 9.039080438e-09, 5.111435690e-10, 1.837124500e-10, 1.079757337e-10],
+        ],
+        [
+            545.57,
+            0.1266,
+            14000,
+            0.4,
+            7.557e-8,
+            4.4e-6,
+            997.0,
+            1060.0,
+            100e-9,
+            [8.776759369e-08, 6.334322297e-09, 2.794904018e-10, 9.255996313e-11, 5.152615472e-11],
+        ],
+    ],
+)
+def test_hydro_spectra(
+    fc,
+    diffusion,
+    f_diode,
+    alpha,
+    gamma0,
+    bead_radius,
+    rho_sample,
+    rho_bead,
+    height,
+    ref_power_spectrum,
+):
+    f = np.array([37.0, 1000.0, 5000.0, 8000.0, 10000.0])
+    power_spectrum = passive_power_spectrum_model_hydro(
+        f,
+        fc=fc,
+        diffusion_constant=diffusion,
+        f_diode=f_diode,
+        alpha=alpha,
+        gamma0=gamma0,
+        bead_radius=bead_radius,
+        rho_sample=rho_sample,
+        rho_bead=rho_bead,
+        distance_to_surface=bead_radius + height,
+    )
+
+    np.testing.assert_allclose(power_spectrum, ref_power_spectrum)
+
+
+@pytest.mark.parametrize(
+    "amp,fc,gamma0,bead_radius,rho_sample,rho_bead,height,ref_power",
+    [
+        [500e-6, 4800, 1.16e-8, 0.52e-6, 997.0, 1060.0, 25e-9, 3.420441139e-11],
+        [250e-6, 4800, 1.16e-8, 0.52e-6, 1030.0, 1060.0, 100e-9, 6.586892852e-12],
+        [500e-6, 4800, 1.26e-8, 0.52e-6, 997.0, 1060.0, 100e-9, 2.636236948e-11],
+        [500e-6, 2445, 1.978234e-8, 1.04e-6, 997.0, 1060.0, 100e-9, 1.178556597e-10],
+        [500e-6, 1165, 4.2694e-8, 2.2e-6, 997.0, 1060.0, 100e-9, 5.576540852e-10],
+        [500e-6, 545.57, 7.557e-8, 4.4e-6, 997.0, 1060.0, 100e-9, 2.467139096e-09],
+    ],
+)
+def test_hydro_spectra(
```

Comment:
is there a reason this has the same name as the previous test function?

Reply:
No, that's an error. Fixing.